css/layout.css

```diff
@@ -0,0 +1,65 @@
+.hidden {
+  display: none;
+}
+
+footer {
+  width: 100%;
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
+ul.nav-bar {
+  margin: 0;
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+li.nav {
+  list-style: none;
+  display: inline;
+}
```

Comment:
These should be in `styles.css`, since they are not layout related

Reply:
Got it.